builtin/mainmenu/init.lua

```diff
@@ -131,7 +140,10 @@ local function init_globals()
 	else
 		tv_main:add(tab_simple_main)
 	end
-	tv_main:add(tab_settings)
+	tv_main:add(tab_simple_settings)
+	if core.setting_getbool("show_advanced_menu") then
```

Comment:
Can the setting be called "show_advanced_settings_menu" ? Otherwise it might lead to confusion.